### PR TITLE
node: only warn on node parameter initialization

### PR DESCRIPTION
**Pull request by haircommander**
because them being errors implies cri-o execution should stop

Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### What type of PR is this?

/kind bug
#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

**Comment by haircommander** [comment]
fixes https://github.com/cri-o/cri-o/issues/5108

File: internal/config/node/node.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func ValidateConfig() error {
 			if i.fatal {
 				return err
 			}
-			logrus.Error(err.Error())
+			logrus.Warnf(err.Error())
```
Comment (saschagrunert) [inline internal/config/node/node.go:80]:
```suggestion
			logrus.Warn(err)
```

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
one unhappy test before all green

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**